development-kit/pkg/engines/jvm/rules.go

```diff
@@ -16,13 +16,40 @@
 package jvm
 
 import (
+	engine "github.com/ZupIT/horusec-engine"
 	"github.com/ZupIT/horusec-engine/text"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/jvm/and"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/jvm/or"
-	"github.com/ZupIT/horusec/development-kit/pkg/enums/engine/advisories/jvm/regular"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/jvm/and"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/jvm/or"
+	"github.com/ZupIT/horusec/development-kit/pkg/engines/jvm/regular"
 )
 
-func AllRulesJvmRegular() []text.TextRule {
+type Interface interface {
+	GetAllRules(rules []engine.Rule) []engine.Rule
+}
+
+type Rules struct{}
+
+func NewRules() Interface {
+	return &Rules{}
+}
+
+func (r *Rules) GetAllRules(rules []engine.Rule) []engine.Rule {
+	for index := range allRulesJvmAnd() {
+		rules = append(rules, allRulesJvmAnd()[index])
```

Comment:
Change for usage item. See commentary in java/rules for more details

Reply:
Fixed